clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThread.java

```diff
@@ -273,9 +301,20 @@ void cleanup() {
             log.error("Unexpected error during shutdown.  Proceed with closing.", e);
         } finally {
             sendUnsentRequests(timer);
+
+            // Copy over the completable events to a separate list, then reap any incomplete
+            // events on that list.
```

Comment:
I think that is clear from the code. We do not need the comment.

Reply:
Removed.